### PR TITLE
Add some competitive aliases

### PR DESCRIPTION
**Pull request by xfix**
Mostly from http://www.smogon.com/dp/articles/pokemon_dictionary

### CODE REVIEW

File: data/aliases.js [hunk 1]
```diff
@@ -236,20 +237,31 @@ exports.BattleAliases = {
 
 	// pokemon
 	"aboma": "Abomasnow",
+	"alaka": "Alakazam",
+	"ape": "Infernape",
```
Comment (TheImmortal) [inline data/aliases.js]: I disagree with this one. nape is fine but this one could mean Primeape.

File: data/aliases.js [hunk 2]
```diff
@@ -236,20 +237,31 @@ exports.BattleAliases = {
...
 	"bliss": "Blissey",
 	"chomp": "Garchomp",
 	"cofag": "Cofagrigus",
+	"conk": "Conkeldurr",
+	"croak": "Toxicroak",
 	"cress": "Cresselia",
 	"dnite": "Dragonite",
 	"don": "Groudon",
 	"dogars": "Koffing",
+	"dos": "Gyarados",
```
Comment (TheImmortal) [inline data/aliases.js]: I don't think I've ever seen dos used.

File: data/aliases.js [hunk 3]
```diff
@@ -236,20 +237,31 @@ exports.BattleAliases = {
...
 	"ekiller": "Arceus",
 	"esca": "Escavalier",
 	"ferro": "Ferrothorn",
+	"ferry": "Ferrothorn",
```
Comment (TheImmortal) [inline data/aliases.js]: lol, forry is used for Forretress but not ferry for Ferrothorn

File: data/aliases.js [hunk 4]
```diff
@@ -236,20 +237,31 @@ exports.BattleAliases = {
...
 	"forry": "Forretress",
 	"gar": "Gengar",
 	"garde": "Gardevoir",
+	"gross": "Metagross",
+	"gyara": "Gyarados",
+	"hax": "Haxorus",
```
Comment (TheImmortal) [inline data/aliases.js]: 50/50 on this one

File: data/aliases.js [hunk 5]
```diff
@@ -257,33 +269,54 @@ exports.BattleAliases = {
 	"landot": "Landorus-Therian",
 	"luke":  "Lucario",
 	"lurk": "Golurk",
+	"mag": "Magnezone",
+	"magna": "Magnezone",
```
Comment (TheImmortal) [inline data/aliases.js]: This should be magn**e**

File: data/aliases.js [hunk 6]
```diff
@@ -257,33 +269,54 @@ exports.BattleAliases = {
...
+	"mamo": "Mamoswine",
 	"mence": "Salamence",
+	"nape": "Infernape",
+	"nite": "Dragonite",
 	"obama": "Abomasnow",
 	"ogre": "Kyogre",
 	"p2": "Porygon2",
+	"phan": "Donphan",
```
Comment (TheImmortal) [inline data/aliases.js]: 50/50 on this one. I don't think I've seen it used but it kinda makes sense.

File: data/aliases.js [hunk 7]
```diff
@@ -257,33 +269,54 @@ exports.BattleAliases = {
...
 
 	// moves
 	"bpass": "Baton Pass",
 	"bp": "Baton Pass",
+	"bu": "Bulk Up",
 	"cc": "Close Combat",
 	"cm": "Calm Mind",
 	"dd": "Dragon Dance",
+	"dm": "Draco Meteor",
+	"dp": "DynamicPunch",
```
Comment (TheImmortal) [inline data/aliases.js]: No.

File: data/aliases.js [hunk 8]
```diff
@@ -257,33 +269,54 @@ exports.BattleAliases = {
...
+	"dpulse": "Dragon Pulse",
 	"eq": "Earthquake",
+	"ep": "Earth Power",
 	"espeed": "ExtremeSpeed",
 	"faintattack": "Feint Attack",
+	"fp": "Focus Punch",
```
Comment (TheImmortal) [inline data/aliases.js]: No. This could mean a lot of things like Fire Punch.

File: data/aliases.js [hunk 9]
```diff
@@ -303,15 +336,22 @@ exports.BattleAliases = {
 	"hpwater": "Hidden Power Water",
 	"hjk": "High Jump Kick",
 	"hijumpkick": "High Jump Kick",
+	"hw": "Heat Wave",
```
Comment (TheImmortal) [inline data/aliases.js]: Not sure about this one since I'm not in the doubles scene.

File: data/aliases.js (hunk 2 above)
Comment (RhythmsAreFromVibrations) [inline data/aliases.js]: (it could mean Zapdos too)

**Comment by RhythmsAreFromVibrations** [comment]
I've seen pory-z and pz used for Porygon-Z, might wanna add

File: data/aliases.js (hunk 2 above)
Comment (TheImmortal) [inline data/aliases.js]: Yeah!

File: data/aliases.js (hunk 4 above)
Comment (ascriptmaster) [inline data/aliases.js]: "hax" is an actual term in Pokemon, so I'm against it.

`haxor` imo lol, and maybe `haxo` but that one's stretching it

**Comment by Zarel** [comment]
+1 for "pory-z" and "pz"

**Comment by ascriptmaster** [comment]
`poryz` and `pz` are already Porygon-Z aliases...

File: data/aliases.js (hunk 2 above)
Comment (CrestfallPS) [inline data/aliases.js]: I've only seen dos used to reference gyarados. The only way I've seen zapdos abbreviated is as zap, which is pretty often.

Comment (RhythmsAreFromVibrations) [inline data/aliases.js]: Regardless, there's still more than Pokemon that can be referred to with "dos"

**Comment by RhythmsAreFromVibrations** [comment]
oh

(I am blind rn brb getting new eyes)

**Comment by ascriptmaster** [comment]
Are you going to get back to this?

**Comment by Zarel** [comment]
@TheImmortal, you decide what to do with this pullreq. I'll probably close in a week if there's no consensus.

**Comment by TheImmortal** [comment]
Doesn't look it's being updated. Not accepting as is.

**Pull request closed without merging**